pkg/controller/util/cluster_util.go

```diff
@@ -56,6 +56,8 @@ const (
 	DefaultClusterHealthCheckTimeout          = 3
 
 	FederationConfigName = "federation-v2"
+	ClusterScope         = "cluster"
```

Comment:
Is it desirable to define these ourselves, or would it make sense to reuse the cluster/namespaced constants from crd?

https://github.com/kubernetes-sigs/federation-v2/blob/master/vendor/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/types.go#L241